pkg/controller/deployment/deployment_controller.go

```diff
@@ -566,19 +566,21 @@ func (dc *DeploymentController) getPodMapForDeployment(d *apps.Deployment, rsLis
 // syncDeployment will sync the deployment with the given key.
 // This function is not meant to be invoked concurrently with the same key.
 func (dc *DeploymentController) syncDeployment(key string) error {
-	startTime := time.Now()
-	klog.V(4).Infof("Started syncing deployment %q (%v)", key, startTime)
-	defer func() {
-		klog.V(4).Infof("Finished syncing deployment %q (%v)", key, time.Since(startTime))
-	}()
-
 	namespace, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
+		klog.ErrorS(err, "Failed to split meta namespace cache key", "key", key)
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to split meta namespace cache key", "cacheKey", key)
```

Reply:
oh, right.
Updated here and above where I copy-pasted that from.